src/gc/gcee.cpp

```diff
@@ -12,6 +12,8 @@
 
 #ifndef DACCESS_COMPILE
 
+#define COUNTER_ONLY(x)
```

Comment:
Could you please make this:

```
#if defined(ENABLE_PERF_COUNTERS)
#define COUNTER_ONLY(x)
#endif
```

Reply:
Actually, all places where this macro is used in this file are inside `ENABLE_PERF_COUNTERS` already, except the first one. So I would just change the following two lines to: 

```
#ifdef ENABLE_PERF_COUNTERS
uint64_t g_TotalTimeInGC = 0;
uint64_t g_TotalTimeSinceLastGCEnd = 0;
#endif // ENABLE_PERF_COUNTERS
```

and delete this macro from this file.